### ISSUE TITLE
Calling "pure" methods without using return value

### ISSUE THREAD

**Issue by aarondandy** [aarondandy/csharp-analysis-tool#19]
```csharp
text.Substring(1, 9);
```

should flag a warning as the code does not mutate test and the return value is ignored
